src/pipeline_manager.py

```diff
@@ -105,10 +107,10 @@ def evaluate(pipeline_name, dev_mode):
 
     y_pred = output['prediction']
 
-    logger.info('Calculating LRMSE on validation set')
-    score = log_root_mean_squared_error(y_true, y_pred)
-    logger.info('LRMSE score on validation is {}'.format(score))
-    ctx.channel_send('LRMSE', 0, score)
+    logger.info('Calculating RMSLE on validation set')
```

Comment:
@kamil-kaczmarek Poteantially some confusion/problems with ol LRMSE channel in neptune